src/rulesets.jl

```diff
@@ -4,11 +4,12 @@ const SIMPLIFY_RULES = RuleSet([
 ])
 
 const NUMBER_RULES = RuleSet([
-    @rule ~t               => ASSORTED_RULES(~t, recurse=false)
-    @rule ~t::is_operation(+) =>  PLUS_RULES(~t, recurse=false)
-    @rule ~t::is_operation(*) => TIMES_RULES(~t, recurse=false)
-    @rule ~t::is_operation(^) =>   POW_RULES(~t, recurse=false)
-    @rule ~t                  =>  TRIG_RULES(~t, recurse=false)
+    @rule ~t                => ASSORTED_RULES(~t, recurse=false)
+    @rule ~t::is_operation(+) =>   PLUS_RULES(~t, recurse=false)
+    @rule ~t::is_operation(*) =>  TIMES_RULES(~t, recurse=false)
+    @rule ~t::is_operation(^) =>    POW_RULES(~t, recurse=false)
+    @rule ~t::is_operation(^) => POWSUM_RULES(~t, recurse=false)
```

Comment:
I think this should be
```suggestion
    @rule ~t::is_operation(+) => POWSUM_RULES(~t, recurse=false)
```
The `is_operation(f)` function tests if a term's outermost `operation` is equal to `f`. For your rules, this will be `+` instead of `^`.